consensus/config/src/parameters.rs

```diff
@@ -20,13 +19,30 @@ pub struct Parameters {
 
     /// The database path. The path should be provided in order for the node to be able to boot
     pub db_path: Option<PathBuf>,
+
+    // Enables pipelining within the universal commit rule.
+    #[serde(default = "Parameters::default_enable_pipelining")]
+    pub enable_pipelining: bool,
```

Comment:
These two variables cannot be in Parameter. As described in the comment above, only values that can tolerate inconsistencies between validators can be in Parameters. Pipelining should always be true and there should not be a parameter for it. `number_of_leaders` per round can be in ProtocolConfig, or a constant for now.